packages/compass-collection/src/components/workspace/workspace.tsx

```diff
@@ -28,9 +28,14 @@ const workspaceViewsStyles = css({
   overflow: 'auto',
 });
 
+// This is relied on in other places in Compass; coupled.
+const workspaceContainerName = 'compass-workspace-container';
```

Comment:
Consider adding to `WorkspaceContainer` in `compass-components` instead, then this can be referenced.